src/renderer/src/components/PromptField.tsx

```diff
@@ -254,9 +254,22 @@ export const PromptField = forwardRef<PromptFieldRef, Props>(
             openModelSelector?.(args);
             break;
           case '/web': {
-            const url = text.replace('/web', '').trim();
+            const commandArgs = text.replace('/web', '').trim();
```

Comment:
Let's use this for simplicity.


```suggestion
            const commandParts = text.split(/\s+/);
```

and then update the rest accordingly.